src/garage/torch/algos/vpg.py

```diff
@@ -294,8 +357,8 @@ def _get_baselines(self, path):
             return torch.Tensor(self.baseline.predict_n(path))
         return torch.Tensor(self.baseline.predict(path))
 
-    def _optimize(self, itr, obs, actions, rewards, valids, baselines):
-        del itr, valids, obs, actions, rewards, baselines
+    def _optimize(self, itr, obs, actions, rewards, baselines, advantages):
+        del itr, obs, actions, rewards, baselines, advantages
```

Comment:
if you don't need these args, why make them part of the function signature?

Reply:
clean up the parameters.
(some of the parameters are used in TRPO)